res/enkf/enkf_simulation_runner.py

```diff
@@ -70,7 +70,7 @@ def runSimpleStep(self, job_queue, run_context):
 
         run_context.get_sim_fs().fsync()
 
-        ## Should be converted tp a looger
+        # TODO: Use logging
```

Comment:
Git blame says this is comment is two years old. Is this still relevant @ManInFez ?

Create an issue if it is relevant. Let's not have TODOs in our code.

Reply:
There are currently 10 TODO comments in the code :D (removed the comment.)